export const getTest = async (req, res) => {
    try {
        console.log('testy');
    } catch (error) {
        console.log(error)
    }
}